fundamentals/problem-set-fundamentals-vocabulary.md

```diff
@@ -580,9 +594,17 @@ Which of the following are literals?
 * `None`
 
 ##### !end-answer
-##### !explanation
+##### !hint
 
+A literal value is a value itself. The value is not referenced by a variable. For example, `3` is a literal integer but `a_number = 3` is not a literal integer.
 
+##### !end-hint
+##### !explanation
+
+- `"Welcome! Make yourselves comfortable!"` is a string literal, `["magenta", "purple", "blue"]` is a list literal, and `None` is a special literal value that represents the absence of a value. These three values are actual values that we might directly write into code. 
```

Comment:
```suggestion
- `"Welcome! Make yourselves comfortable!"` is a string literal, `["magenta", "purple", "blue"]` is a list literal, and `None` is a special literal that represents the absence of a value. These three values are actual values that we might directly write into code. 
```
Just a small suggestion to avoid repetition.